jobboard/models.py

```diff
@@ -47,11 +47,20 @@ class Job(models.Model):
     title_keywords = models.ManyToManyField(JobTitleKeyword)
 
     @classmethod
-    def get_jobs_by_city_name(cls, city_name):
+    def get_jobs_by_city_name(cls, city_names_as_string):
         """
         Allows students to search for a job based on the city it is located in
         """
-        return set(cls.objects.filter(city=City.objects.get(name=city_name)))
+        jobs_to_return = set()
+        city_names = city_names_as_string.split()
```

Comment:
I think splitting the string like this might cause issues if the city name is not one word like "Tel Aviv" for example.

Reply:
Yeah, I agree with @guyy13 . I think we should leave that function the way it was

Reply:
@yaelerdr 
That's a bug, if I search for Tel Aviv I want to see jobs in Tel Aviv, not in Tel Mond.

Reply:
I agree this can be a bit problematic however I can't think of a better way to implement this. The user won't necessarily type "Tel-Aviv" in the search bar and if he types "Tel Aviv" then the way you guys are suggesting won't return any of the Tel-Aviv jobs. I think displaying jobs that the user doesn't specifically search for (like Tel Mond when searched for Tel-Aviv)  in addition to the ones he did search for is better then displaying 0 jobs because he didn't add a "-". Also if I return the model function to the way it was before, the user can only search 1 city at a time and I think its good that he can search a bunch of cities at once. 
But I'm open to suggestions if you guys can think of a better way to implement this.

Reply:
@eldaraharonson We can maybe add the function of a complete match to a city in DB, run this one according to the user's search, and if there wasn't a perfect match, then run the function of partially matched results.